### ISSUE TITLE
When you misspell someone's name you can't press back.

### ISSUE THREAD

**Issue by MariusHendriks** [kevinclement/SimpleArmory#315]
When you type someone's name in the url, and it does not exist or you made typo, it takes you to a different page. This is really annoying because you just want to change the misspelled letter, but instead you are on some error page.

**Comment by finnlux** [comment]
I don't see this issue. I get taken to a page that says Click here to try again. Either this was fixed already or could you describe it better?

**Comment by MariusHendriks** [comment]
Hi Finnlux! Thanks for comming back to me!

I will discribe it as i  experience it. 

1. I go to my page: https://simplearmory.com/#/eu/twisting-nether/jezuss/collectable/mounts
2. Then i change the link to that of my friend: https://simplearmory.com/#/eu/twisting-nether/digitzz/collectable/mounts
3. I get an error because either i made a typo or his account does not exist anymore. 
4. I want to head back to my own page by pressing the back arrow of the browser
5. I get stuck in a error screen instead of my own page. 


I hope this is clear enough. It's not a real breaking-problem but it is something that has annoyed me a couple times.

Take care and thanks for the website! I use it more then google ;).

Cheers!

**Comment by kevinclement** [comment]
This was pretty annoying and had to do with how we were handling url change notifications.  When you hit back your url would actually change to the "bad user" again, and then forward you again to the error page.  Lame.

I just deployed the framework change to svelte tonight.  I discovered this bug while doing that work and fixed it so this should be fixed on the live site now.  

I also fixed one more edge case too where it's possible that the user sees the error page and then changes the current url to "fix" the bad character.  If the user doesn't also remove the '/error' on the front of the url it won't do what she is expecting.  So I detect that and put them back into the "login" state.

Let me know if you don't see it fixed.  Cheers.
